generators/client/templates/angular/src/main/webapp/content/scss/global.scss.ejs

```diff
@@ -35,6 +35,10 @@ h4 {
     font-weight: 300;
 }
 
+.container-fluid {
+  padding-top: 45px;
```

Comment:
I'm not a CSS specialist, but is not a good idea to use px to set a padding value?